clio-client/src/main/scala/org/broadinstitute/clio/client/commands/DeleteWgsUbamCommand.scala

```diff
@@ -0,0 +1,145 @@
+package org.broadinstitute.clio.client.commands
+
+import akka.http.scaladsl.model.HttpResponse
+import com.typesafe.scalalogging.{LazyLogging, Logger}
+import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport
+import org.broadinstitute.clio.client.parser.BaseArgs
+import org.broadinstitute.clio.client.util.{FutureWithErrorMessage, IoUtil}
+import org.broadinstitute.clio.client.webclient.ClioWebClient
+import org.broadinstitute.clio.transfer.model.{
+  TransferWgsUbamV1Key,
+  TransferWgsUbamV1Metadata,
+  TransferWgsUbamV1QueryInput,
+  TransferWgsUbamV1QueryOutput
+}
+import org.broadinstitute.clio.util.json.ModelAutoDerivation
+import org.broadinstitute.clio.util.model.{DocumentStatus, Location}
+import org.broadinstitute.clio.client.util.WgsUbamUtil.TransferWgsUbamV1QueryOutputUtil
+
+import scala.collection.immutable.Seq
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success}
+
+object DeleteWgsUbamCommand
+    extends Command
+    with LazyLogging
+    with FailFastCirceSupport
+    with ModelAutoDerivation
+    with FutureWithErrorMessage {
+
+  implicit val implicitLogger: Logger = logger
+
+  override def execute(
+    webClient: ClioWebClient,
+    config: BaseArgs,
+    ioUtil: IoUtil
+  )(implicit ec: ExecutionContext): Future[HttpResponse] = {
+    for {
+      _ <- webClient.verifyCloudPaths(config) withErrorMsg ()
+      queryResponses <- webClient
+        .queryWgsUbam(
+          config.bearerToken.get,
+          TransferWgsUbamV1QueryInput(
+            flowcellBarcode = config.flowcell,
+            lane = config.lane,
+            libraryName = config.libraryName,
+            location = Location.pathMatcher.get(config.location.get),
+            documentStatus = Some(DocumentStatus.Normal)
+          )
+        )
+        .map(webClient.ensureOkResponse) withErrorMsg "There was a problem querying the Clio server for WgsUbams."
+      wgsUbams <- webClient.unmarshal[Seq[TransferWgsUbamV1QueryOutput]](
+        queryResponses
+      )
+      deleteResponses <- deleteWgsUbams(wgsUbams, webClient, config, ioUtil)
+    } yield {
+      deleteResponses.size match {
+        case s if s != deleteResponses.size =>
```

Comment:
I'm pretty sure this `if` will never be true, since you're matching on `deleteResponses.size`. I'm also not sure if distinguishing the 0 case is worth much.

Could we change the match to be like:
```scala
deleteResponses.size match {
  case s if s == wgsUbams.size => OK
  case _ => Not OK
}
```